shared/@hedvig-ui/TextDatePicker/text-datepicker.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import { TextDatePicker } from '@hedvig-ui'
+import React from 'react'
+
+export default {
+  title: 'TextDatePicker',
+  component: TextDatePicker,
+}
+
+export const StandardTextDatePicker: React.FC = () => {
+  const [error, setError] = React.useState(false)
+  const [value, setValue] = React.useState<Date | null>()
+
+  const setDateHandler = (date: Date | null) => {
+    if (date) {
+      setError(false)
+      setValue(date)
+    } else {
+      setError(true)
+      setValue(null)
+    }
+  }
+
+  return (
+    <div style={{ padding: 30 }}>
```

Comment:
Could also use the `Spacing` component from `hedvig-ui`
```suggestion
    <Spacing top>
```